tests/loading_from_clipboard/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+ADD_DEFINITIONS("-DCARDDB_DATADIR=\"${CMAKE_CURRENT_SOURCE_DIR}/data/\"")
+add_executable(loading_from_clipboard_test
+        loading_from_clipboard_test.cpp
+        ../../common/decklist.cpp
+        )
+
+if(NOT GTEST_FOUND)
+    add_dependencies(loading_from_clipboard_test gtest)
+endif()
+
+target_link_libraries(loading_from_clipboard_test ${GTEST_BOTH_LIBRARIES})
+target_link_libraries(loading_from_clipboard_test cockatrice_common)
+
+add_test(NAME loading_from_clipboard_test COMMAND loading_from_clipboard_test)
+
+#### I feel like the rest of this file should not be necessary and
+#### is (effective) cargo culting of tests/carddatabase/CMakeLists.txt.
+#### Ideally we would only need to say "hey this test is against something from cockatrice_common",
+#### and cockatrice_common would declare all of it's dependencies. I need to learn more about CMake.
```

Comment:
I agree this should be simpler to implement.
If you want to both fix this an take the "single test binary" road, we could create a single test executable that includes the whole cockatrice code and all the cpp from the tests/ directory, replacing cockatrice's `main()` method with a mockup that just runs all the tests.